addon/modifiers/did-update.js

```diff
@@ -1,5 +1,6 @@
 import { setModifierManager, capabilities } from '@ember/modifier';
 import { gte } from 'ember-compatibility-helpers';
+import { untrack } from '@glimmer/validator';
```

Comment:
I don't see how this import will work for ember-source < 3.27, in versions prior to 3.27 these sorts of internal ember modules were hidden and not available for import...

Reply:
I approved the CI run, so we should see those failures (in 3.24 at least)

Reply:
```
not ok 5 Chrome 96.0 - [38 ms] - Integration | Modifier | did-update: it basically works
    ---
        stack: >
            Error: Could not find module `@glimmer/validator` imported from `@ember/render-modifiers/modifiers/did-update`
                at missingModule (http://localhost:7357/assets/vendor.js:259:11)
                at findModule (http://localhost:7357/assets/vendor.js:270:7)
                at Module.findDeps (http://localhost:7357/assets/vendor.js:180:24)
                at findModule (http://localhost:7357/assets/vendor.js:274:11)
                at Module.findDeps (http://localhost:7357/assets/vendor.js:180:24)
                at findModule (http://localhost:7357/assets/vendor.js:274:11)
                at requireModule (http://localhost:7357/assets/vendor.js:36:15)
                at ModuleRegistry.get (http://localhost:7357/assets/vendor.js:67734:14)
                at Class._extractDefaultExport (http://localhost:7357/assets/vendor.js:68170:41)
                at Class.resolveOther (http://localhost:7357/assets/vendor.js:67830:32)
```

Reply:
I've pushed a commit that should resolve this issue...